### PR TITLE
change computation ULs to OLs

### LINKED ISSUE #185: Use Consistent Numbering scheme in Computation Steps
The numbering schemes for the sub-steps of [4.3.2](https://w3c.github.io/accname/#computation-steps) are insistent from step to step. In step 2.B uses bullets then roman numerals and then letters, while steps 2.A, 2.C, and 2.D use bullets throughout. Step 2.F uses roman numerals and then bullets or letters. This makes the algorithm more difficult to parse than it needs to be, and given that all the steps should be executed in sequence, they should probably all be ordered lists.

### PR DESCRIPTION
**Pull request by cookiecrook**
resolves #185

pls merge #187 first, as this one branched from it

### CODE REVIEW

**Comment by cookiecrook** [comment]
@spectranaut I noticed when starting this new PR that the template completely overwrites the multi-line commit message. It used to append the template to the commit message. What that a purposeful change? Since ~last week IIRC.

File: index.html [hunk 1]
```diff
@@ -355,31 +355,31 @@ <h4>Computation steps</h4>
...
                   <ol>
                     <li id="comp_labelledby_reset" name="step2B.i">Set the <code>accumulated text</code> to the empty string.</li>
                     <li id="comp_labelledby_foreach" name="step2B.ii">For each IDREF:
                       <ol>
                         <li id="comp_labelledby_set_current" name="step2B.ii.a">Set the <code>current node</code> to the node referenced by the IDREF.</li>
                         <li id="comp_labelledby_recursion" name="step2B.ii.b"><em>LabelledBy Recursion:</em> Compute the text alternative of the <code>current node</code> beginning with the overall <a href="#comp_computation">Computation</a> step. Set the <code>result</code> to that text alternative.</li>
-                        <li id="comp_labelledby_append" name="step2B.ii.c">Append the <code>result</code>, with a space, to the <code>accumulated text</code>.</li>
+                        <li id="comp_labelledby_append" name="step2B.ii.c">Append a space character and the <code>result</code> to the <code>accumulated text</code>.</li>
```
Comment (cookiecrook) [inline index.html:366]: Note that this is from #187, not #188.

File: index.html [hunk 2]
```diff
@@ -464,11 +464,11 @@ <h4>Computation steps</h4>
                 <p>Tooltip attributes are used only if nothing else, including subtree content, has provided results. </p>
               </details></div>
             </li>
+            <li id="comp_append">Append a space character and the <code>result</code> of each step above to the <code>total accumulated text</code>.</li>
```
Comment (cookiecrook) [inline index.html:467]: All remaining diffs are from #187, not #188.

Comment (cookiecrook) [reply index.html:467]: But those will disappear from this diff once PR #187 is merged.

**Comment by spectranaut** [comment]
> @spectranaut I noticed when starting this new PR that the template completely overwrites the multi-line commit message. It used to append the template to the commit message. What that a purposeful change? Since ~last week IIRC.

I tried to see if you could preserve the commit messages while using a PR template, but there doesn't seem to be an option!

File: index.html [hunk 3]
```diff
@@ -391,11 +391,11 @@ <h4>Computation steps</h4>
                 <li id="comp_embedded_control_textbox"><em>Textbox:</em> If the embedded control has role <a class="role-reference" href="#textbox">textbox</a>, return its value.</li>
```
Comment (spectranaut) [inline index.html:391]: This list is an "unorder list", and looks odd, given everything else is ordered now

**Review by spectranaut — CHANGES_REQUESTED** [review]

**Review by spectranaut — APPROVED** [review]

**Review by MelSumner — APPROVED** [review]

**Comment by cookiecrook** [comment]
@jnurthen @spectranaut merge #187 then this one? thx

**Pull request merged**